data/config/bhc.json

```diff
@@ -70,7 +70,6 @@
       ]
     },
     "navigation": {
-      "title": "BRAIN Commons",
```

Comment:
why this change?

Reply:
Because the logo has the title in it so just going to use that as the nav title.
Similar to NIAID!